docs/pages/access-controls/idps/saml.mdx

```diff
@@ -0,0 +1,111 @@
+
+```yaml
+kind: saml_idp_service_provider
+version: v1
+metadata:
+  # The friendly name of the service provider. This is used to manage the
+  # service provider as well as in identity provider initiated SSO.
+  name: saml-service-provider
+spec:
+  # The entity_descriptor is the service provier XML.
+  entity_descriptor: |
+    <md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"...
+```
+
+Add this to Teleport using `tctl`:
+
+```code
+$ tctl create saml-sp.yaml
+# SAML IdP service provider 'saml-service-provider' has been created.
+```
+
+With this, Teleport now trusts the service provider.
+
+### Configure the application for SAML SSO
+
+The first step in configuring the application for SSO is retrieving Teleport's
+Teleport's SAML identity provider metadata. You can obtain this metadata in XML
+format by navigating to `https://<proxy-address>/enterprise/saml-idp/metadata`. Save
+it in an easy to remember file name like `teleport-metadata.xml`.
+
+Next, you'll need to follow your application's documentation on how to configure
+the application for SSO. This will vary from application to application, but you
+may need to know the SAML assertions that the Teleport identity provider currently
+provides and specify how they map into permissions and identity within the
+application. At present, these assertions are:
+
+| Friendly name | Full name | Description | Format |
+|---------------|-----------|-------------|--------|
+| `uid` | `urn:oid:0.9.2342.19200300.100.1.1` | The user ID from Teleport | `urn:oasis:names:tc:SAML:2.0:attrname-format:uri` |
+| `eduPersonAffiliation` | `urn:oid:1.3.6.1.4.1.5923.1.1.1.1` | The user's Teleport roles | `urn:oasis:names:tc:SAML:2.0:attrname-format:uri` |
+
+Additionally, you will need to upload Teleport's identity provider metadata into
+the application. Please refer to the application's documentation on how to do
+this using the previously retrieved `teleport-metadata.xml`.
+
+<Admonition type="tip">
+  Many applications will allow you to upload the identity provider metadata
+  directly, but some may require you to parse information out of the XML
+  and add it to the application piecemeal. The application documentation will
+  assist with this.
+</Admonition>
+
+Once the application is properly configured, you are ready to use Teleport as a
+SAML identity provider.
+
+## Service provider/identity provider initiated SSO
+
+The Teleport SAML identity provider supports both service provider initiated and
+identity provider initiated SSO. Service provider initiated SSO is simple: attempt
+to log in from the configured external application. To trigger identity provider
+initiated SSO, navigate to
+`https://<proxy-address>/enterprise/saml-idp/login/<sp-name>` where
+`sp-name` is the name of the service provider in Teleport. For the example above,
+assuming our proxy address is `example.com`, the URL would be
+`https://example.com/enterprise/saml-idp/login/saml-service-provider`.
```

Comment:
After this sentence, might be worth mentioning what actually happens if you go directly to this URL.